_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,185 @@
+function SlashedPrice() {
+	return (
+		<div className="slashed-price__container" style={ { marginRight: '14px' } }>
+			<div className="slashed-price__slash"></div>
+			{ /* TODO: get this from an API or calculate, currently unsure how to get this increased price */ }
+			<div className="slashed-price__price">{ '$15-25' }</div>
+		</div>
+	);
+}
+
+function SingleProductBackupBody( props ) {
+	const { selectedBackupType, setSelectedBackupType, sitePlans } = props;
+
+	const upgradeLinks = {
+		'real-time': 'https://wordpress.com/jetpack/connect/pro',
+		daily: 'https://wordpress.com/jetpack/connect/premium',
+	};
+
+	const upgradeTitles = {
+		'real-time': __( 'Upgrade to Real-Time Backups' ),
+		daily: 'Upgrade to Daily Backups',
+	};
+
+	function handleSelectedBackupTypeChange( event ) {
+		setSelectedBackupType( event.target.value );
+	}
+
+	return (
+		<div className="plans-section__body">
+			<p>
+				{ __(
+					'Always-on backups ensure you never lose your site. Choose from real-time or daily backups. {{a}}Which one do I need?{{ext/}}{{/a}}',
+					{
+						components: {
+							a: <a href="https://jetpack.com/upgrade/backup/" />,
+							ext: (
+								<>
+									<span>
+										<Gridicon icon="external" size="12" />
+									</span>
+								</>
+							),
+						},
+					} // TODO: make icon color change with link
+				) }
+			</p>
+
+			<h4>Backup options:</h4>
+			<div
+				style={ {
+					display: 'flex',
+					flexDirection: 'row',
+					justifyContent: 'center',
+				} }
+			>
+				<PlanRadioButton
+					planName={ __( 'Daily Backups' ) }
+					radioValue={ 'daily' }
+					planPrice={ sitePlans && sitePlans[ 'daily-backup' ].price.yearly.text }
+					checked={ 'daily' === selectedBackupType }
+					onChange={ handleSelectedBackupTypeChange }
+				/>
+				<PlanRadioButton
+					planName={ __( 'Real-Time Backups' ) }
+					radioValue={ 'real-time' }
+					planPrice={ sitePlans && sitePlans[ 'realtime-backup' ].price.yearly.text }
+					checked={ 'real-time' === selectedBackupType }
+					onChange={ handleSelectedBackupTypeChange }
+				/>
+			</div>
+
+			<div style={ { textAlign: 'center', marginTop: '23px', marginBottom: '10px' } }>
+				<a href={ upgradeLinks[ selectedBackupType ] } type="button" class="dops-button is-primary">
+					{ upgradeTitles[ selectedBackupType ] }
+				</a>
+			</div>
+		</div>
+	);
+}
+
+export function PlanRadioButton( props ) {
+	const { checked, onChange, planName, radioValue, planPrice } = props;
+
+	return (
+		<div className="plan-radio-button__container">
+			<div style={ { gridColumn: 1, gridRow: 1 } }>
+				<input
+					type="radio"
+					style={ { marginRight: '10px' } }
+					value={ radioValue }
+					checked={ checked }
+					onChange={ onChange }
+				/>
+			</div>
+			<div style={ { gridColumn: 2, gridRow: 1, fontWeight: 'bold' } }>{ planName }</div>
+			<div style={ { gridColumn: 2, gridRow: 2 } }>
+				{ /* TODO: how to I18N this? */ }
+				{ planPrice && `${ planPrice } /year` }
```

Comment:
We typically pull this dynamically, like here:
https://github.com/Automattic/jetpack/blob/3f1a188fa6bfdb5a48088f26e8da5da4a01e92bd/_inc/client/plans/plan-grid.jsx#L253

Reply:
I did look at that since other areas do the same thing, but it's difficult to get the "/year" in there. I'm thinking I may need to update that server endpoint to send html with the "/year".